### PR TITLE
improved build from source

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
Improves build from source docs

### Motivation
ambition

### Preview link

https://docs-staging.datadoghq.com/cswatt/build-from-source/agent/basic_agent_usage/source

### CODE REVIEW

File: content/en/agent/basic_agent_usage/source.md [hunk 1]
```diff
@@ -21,6 +21,31 @@ This page outlines the basic features of the Datadog Agent. If you haven't insta
 
 By default, your Agent is installed in its own sandbox at `~/.datadog-agent`. You're free to move this folder wherever you like. However, this article assumes that the Agent is installed in its default location, so be sure to modify the instructions accordingly if you decide to move them.
 
+## Linux Installation
+
+* Install [Go][2] version 1.11.5 or later. Remember to set your `GOPATH` and have `$GOPATH/bin` in your path.
+* Install Python version 3.x, along with development libraries.
+* Install CMake version 3.12+.
+* In your virtual environment, install Python dependencies. You may use `pip install -r requirements.txt`
```
Comment (burningion) [inline content/en/agent/basic_agent_usage/source.md]: You create a virtual environment in Python, and this should be below the `Install Python` line.

Activate your virtual environment, and install the Python dependencies with a `pip install -r requirements.txt`.

File: content/en/agent/basic_agent_usage/source.md [hunk 2]
```diff
@@ -21,6 +21,31 @@ This page outlines the basic features of the Datadog Agent. If you haven't insta
...
+* Checkout the repo: `git clone https://github.com/DataDog/datadog-agent.git $GOPATH/src/github.com/DataDog/datadog-agent`
+* Install the project's dependencies with `invoke deps`. Ensure that `$GOPATH/bin` is in your `$PATH`.
```
Comment (burningion) [inline content/en/agent/basic_agent_usage/source.md]: You can get rid of the `Ensure that GOPATH` line here, you've already verified it above.

**Comment by olivielpeau** [comment]
Unfortunately this information on this page is a bit outdated:

The source install only really exists for the Agent v5. For Agent v6 IMO we should point to the "developer docs" of the datadog-agent repo which contains information on how to build the Agent binaries and packages from the source: https://github.com/DataDog/datadog-agent/blob/master/docs/dev/README.md (which is linked from the main README of the repo: https://github.com/DataDog/datadog-agent#getting-started).

Happy to discuss how to make these developer docs easier to understand if needed. As these instructions can change quite a lot between Agent versions I think it's important that we keep them in `datadog-agent` though (and for now only link to them from the website), so that we can update these docs as we make changes to the way the Agent is built.

**Comment by cswatt** [comment]
Closing the PR, will revisit this after agent core works on the OKR pertaining to this.

**Pull request closed without merging**